test/test_autograd.py

```diff
@@ -2789,15 +2789,14 @@ def test_cat_empty(self):
                               lambda a, b: torch.cat((a, b)),
                               True, f_args_variable, f_args_tensor, check_forward_ad=True)
 
-    def test_trapz(self):
+    def test_trapezoid(self):
```

Comment:
Is this test redundant with the automatically generated tests in test_ops.py? Can it be removed?

Reply:
Yes I believe it is redundant. Removed.